cmd/module.go

```diff
@@ -47,9 +53,35 @@ func init() {
 		"The name of the module")
 	buildModuleCmd.Flags().StringVarP(&buildModuleCmdPlatform, "platform", "p", "cf",
 		`The deployment platform; supported platforms: "cf", "xsa", "neo"`)
+
+	// sets the flags of the solo Makefile command build module
+	soloBuildModuleCmd.Flags().StringVarP(&soloBuildModuleCmdSrc, "source", "s", "",
+		"The path to the MTA project; the current path is set as default")
+	soloBuildModuleCmd.Flags().StringVarP(&soloBuildModuleCmdTrg, "target", "t", "",
+		"The path to the folder in which the module build results are created; the <current folder>/.<project name>_mta_build_tmp/<module name> path is set as default")
+	soloBuildModuleCmd.Flags().StringSliceVarP(&soloBuildModuleCmdExtensions, "extensions", "e", nil,
+		"The MTA extension descriptors")
+	soloBuildModuleCmd.Flags().StringVarP(&soloBuildModuleCmdModule, "module", "m", "",
+		"The name of the module")
+}
+
+// soloBuildModuleCmd - Build module command used stand along
+var soloBuildModuleCmd = &cobra.Command{
+	Use:   "module-build",
+	Short: "Builds module",
+	Long:  "Builds module and archives its artifacts",
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := artifacts.ExecuteSoloBuild(buildModuleCmdSrc, soloBuildModuleCmdTrg, soloBuildModuleCmdExtensions, soloBuildModuleCmdModule, os.Getwd)
```

Comment:
It should be `soloBuildModuleCmdSrc` instead of `buildModuleCmdSrc`

Reply:
corrected